src/app/home/avatar-generator/selector/atribute-selector.scss

```diff
@@ -0,0 +1,24 @@
+@import '../../../../styles.variables.scss';
+
```

Comment:
Please add the "component" suffix to the name of this file: attribute-selector.component.scss

Reply:
fixed

Reply:
The "component" suffix is still missing here... Also, if you can fix the type on "atribute" too, it should be "attribute" with double 't'